mods/extrahorns/init.lua

```diff
@@ -0,0 +1,66 @@
+-- large horn
+minetest.register_craftitem ("extrahorns:largehorn", {
+	description = "Large Horn",
+	inventory_image = "extrahorns_largehorn.png",
+	groups = {instrument=1},	
+	on_use = function (itemstack, user)
+		minetest.sound_play ("extrahorns_largehorn", {
+			pos = user:getpos(),
+			max_hear_distance = 450,
+			gain = 1,
+		})
+	end,
+})
+
+minetest.register_craft ({
+	type   = "shapeless",
+	output = "extrahorns:largehorn",
+	recipe = {"soundblocks:smallhorn", "default:steel_ingot"}
+})
+
+-- Warhorn
+
+minetest.register_craftitem ("extrahorns:warhorn", {
+	description = "War Horn" ,
+	inventory_image = "extrahorns_warhorn.png",
+	groups = {instrument=1},	
+	on_use = function (itemstack, user)
+		minetest.sound_play ("extrahorns_warhorn", {
+			pos= user:getpos() ,

```

Comment:
Inconsistent spacing. Please also replace all occurrences of `getpos` with `get_pos` as the former is deprecated.

```lua
pos = user:get_pos(),
```